ui/src/pages/backoffice/Acquisition/Order/OrderDetails/OrderDetails.js

```diff
@@ -129,25 +122,19 @@ class OrderPanels extends React.Component {
           Order information
         </Header>
         <Segment attached className="bo-metadata-segment">
-          <div ref={this.orderTopRef} id="order-info">
-            <OrderInformation order={data.metadata} />
-          </div>
+          <OrderInformation order={data.metadata} />
```

Comment:
Do we need the complete metadata in these components? Can we pass a smaller payload?

Reply:
It is a bit not a scope of this PR
but I would rather pass the whole metadata and leave choosing the specific fields of it to the component itself, not the parent. IMO it is cleaner